pkgs/by-name/qm/qmplay2/package.nix

```
@@ -0,0 +1,113 @@
+{
+  lib,
+  alsa-lib,
+  callPackage,
+  cmake,
+  ffmpeg,
+  fribidi,
+  game-music-emu,
+  libXdmcp,
+  libXv,
+  libass,
+  libcddb,
+  libcdio,
+  libpulseaudio,
+  libsidplayfp,
+  libva,
+  libxcb,
+  ninja,
+  pkg-config,
+  qt5,
+  qt6,
+  stdenv,
+  taglib,
+  vulkan-headers,
+  vulkan-tools,
+  # Configurable options
+  qtVersion ? "6", # Can be 5 or 6
+}:
+
+let
+  sources = callPackage ./sources.nix { };
+in
+assert lib.elem qtVersion [ "5" "6" ];
+stdenv.mkDerivation (finalAttrs: {
+  pname = sources.qmplay2.pname + "-qt" + qtVersion;
```

Non-blocking nit: if I were writing this, I'd use
```suggestion
  pname = "${sources.qmplay2.pname}-qt${qtVersion}";
```